docs/release-notes/highlights-1.3.0.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[release-highlights-1.3.0]]
+== 1.3.0 release highlights
+
+[float]
+[id="{p}-130-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 1.3.0 of {n}. See <<release-notes-1.3.0>> for the full list of changes.
+
+[float]
+[id="{p}-130-official-helm-chart"]
+==== Official Helm chart
+
+Addressing one of the most common asks, ECK 1.3.0 introduces <<{p}-install-helm,an official Helm chart>> which allows to configure and deploy ECK. Apart from enabling new deployments, a migration path that enables moving from previous deployment method to a Helm chart is documented.
```

Comment:
ECK 1.3.0 introduces an official Helm chart for deploying the operator. This provides a more flexible way of installing the operator customized to the environment and the desired use case. Existing installations can be easily migrated to Helm as well.